crates/vm/levm/Makefile

```
@@ -18,28 +18,44 @@ fmt: ## 📄 Runs rustfmt
 EFTEST_DIR := ../../../cmd/ef_tests/levm
 VECTORS_DIR := $(EFTEST_DIR)/vectors
 TMP_DIR := tmp
+TESTS_REPO := $(TMP_DIR)/ethereum-tests
 
-SPECTEST_VERSION := v14.1
-SPECTEST_ARTIFACT := tests_$(SPECTEST_VERSION).tar.gz
-SPECTEST_URL := https://github.com/ethereum/tests/archive/refs/tags/$(SPECTEST_VERSION).tar.gz
+ETH_TEST_URL := https://github.com/ethereum/tests.git
+ETH_TEST_TAG := v14.1
+COMMIT_LEGACY_TESTS_FOR_TAG := b2e6c9e
 
 STATETEST_VERSION := pectra-devnet-5%40v1.1.0
 STATETEST_NET := pectra-devnet-5
 STATETEST_ARTIFACT := fixtures_$(STATETEST_NET).tar.gz
 STATETEST_URL := https://github.com/ethereum/execution-spec-tests/releases/download/$(STATETEST_VERSION)/fixtures_$(STATETEST_NET).tar.gz
 
-download-evm-ef-tests: ## 📥 Download EF Tests
-	mkdir -p $(TMP_DIR)
+setup-test-dirs:
+	mkdir -p $(VECTORS_DIR)
+	mkdir -p $(VECTORS_DIR)/LegacyTests/Constantinople/GeneralStateTests
 	mkdir -p $(VECTORS_DIR)/GeneralStateTests
 	mkdir -p $(VECTORS_DIR)/state_tests
-	curl -L -o $(SPECTEST_ARTIFACT) $(SPECTEST_URL)
-	tar -xzf $(SPECTEST_ARTIFACT) -C $(TMP_DIR)
-	mv $(TMP_DIR)/tests-14.1/GeneralStateTests/* $(VECTORS_DIR)/GeneralStateTests/
+
+clone-ef-tests: ## 📥 Download Ethereum Tests repository with submodules
+	mkdir -p $(TMP_DIR)
+	git clone --recurse-submodules $(ETH_TEST_URL) $(TESTS_REPO) 
+	cd $(TESTS_REPO) && git checkout tags/$(ETH_TEST_TAG)
```

This can be simplified to:
```suggestion
	git clone --recurse-submodules -b tags/$(ETH_TEST_TAG) $(ETH_TEST_URL) $(TESTS_REPO)
```
I believe that would also cover the following line, but check.

---

Thanks! I tried that but it didn't work out. I stick to:
`git clone --recurse-submodules --depth 1 --branch $(ETH_TEST_TAG) $(ETH_TEST_URL) $(TESTS_REPO)`
Done here [16a6725](https://github.com/lambdaclass/ethrex/pull/1870/commits/16a672575ba27c60b35e87be2a7053b8751b0194)